buildconfig/CMake/WindowsNSIS.cmake

```diff
@@ -144,50 +162,39 @@ install ( FILES ${QT_PLUGIN_DIR}/sqldrivers/qsqlite4.dll DESTINATION plugins/qt4
 ###########################################################################
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/buildconfig/CMake/Packaging/launch_mantidplot.bat DESTINATION bin )
 install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/buildconfig/CMake/Packaging/launch_mantidplot.vbs DESTINATION bin )
-install ( FILES ${PROJECT_BINARY_DIR}/mantidpython.bat.install DESTINATION bin 
-    RENAME mantidpython.bat )
-
-###########################################################################
-# Icons for shortcuts
-###########################################################################
-install ( FILES ${CMAKE_CURRENT_SOURCE_DIR}/images/mantid_notebook.ico DESTINATION bin )
```

Comment:
Do we not need to keep the `mantid_notebook.ico` now?

Reply:
The icon has found its way back to Mantid 🎉